app/NavEntry.js

```diff
@@ -29,6 +34,27 @@ class MainNavigation extends React.Component {
     }
   }
 
+  async componentDidMount() {
+    const { t } = this.props;
+    try {
+      const { data } = await fetch(BULLETINS_URL);
+      const bulletins = data.bulletins;
```

Comment:
You can delete the bulletins constant variable and destructuring from data like this
```suggestion
      const { data: {bulletins} } = await fetch(BULLETINS_URL);
```